__tests__/app.test.js

```diff
@@ -241,6 +241,10 @@ describe('Articles', () => {
 
     }); // Describe: GET /api/article/:article_id
 
+}); // Describe: Articles
```

Comment:
You could/should make this another describe block instead of commenting it

Reply:
I've just been looking over the source file; and I believe this is the end of the Articles describe block. I usually include comments at the end of blocks (when there are multiple of them nested across a large file) as I find it easier to know which are ending & where. But given your comment, I understand that it must seem a little bit confusing for others; I'll consider changing these to something like:

```
// Ending: Describe Articles
```